Add mutation percentage info when success

Hey,

While using maven, I run the mutation testing using the command specified in the docs:

```
mvn org.pitest:pitest-maven:mutationCoverage
```

if I specify a certain [mutation threshold](http://pitest.org/quickstart/maven/#mutationthreshold) that is not met,  I get an error message on the command line:

```
[ERROR] Failed to execute goal org.pitest:pitest-maven:1.4.0:mutationCoverage (default-cli) on project <XXXXX>: Mutation score of 85 is below threshold of 100 -> [Help 1]
```

and this is simple but good enough.  But I have no similar output when the score is above the threshold. As far as I've seen, there's no option/configuration for that. Can you confirm? 

Looking at the source code, if I wanted to implement this, it would be rather easy no?
It seems I would only need to add a simple log at https://github.com/hcoles/pitest/blob/master/pitest-maven/src/main/java/org/pitest/maven/AbstractPitMojo.java?utf8=%E2%9C%93#L378-L382. And for consistency, probably also add to https://github.com/hcoles/pitest/blob/master/pitest-command-line/src/main/java/org/pitest/mutationtest/commandline/MutationCoverageReport.java#L42-L51 no?

Note: using version 1.4.0

--- a/pitest-entry/src/main/java/org/pitest/mutationtest/tooling/MutationCoverage.java
+++ b/pitest-entry/src/main/java/org/pitest/mutationtest/tooling/MutationCoverage.java
@@ -225,6 +225,15 @@ private int numberOfThreads() {
 
   private void printStats(final MutationStatisticsListener stats) {
     final PrintStream ps = System.out;
+
+    ps.println(StringUtil.separatorLine('='));
+    ps.println("- Mutators");
+    ps.println(StringUtil.separatorLine('='));
+    for (final Score each : stats.getStatistics().getScores()) {
+      each.report(ps);
+      ps.println(StringUtil.separatorLine());
+    }
+
     ps.println(StringUtil.separatorLine('='));
     ps.println("- Timings");
     ps.println(StringUtil.separatorLine('='));
@@ -234,14 +243,6 @@ private int numberOfThreads() {
     ps.println("- Statistics");
     ps.println(StringUtil.separatorLine('='));
     stats.getStatistics().report(ps);
-
-    ps.println(StringUtil.separatorLine('='));
-    ps.println("- Mutators");
-    ps.println(StringUtil.separatorLine('='));
-    for (final Score each : stats.getStatistics().getScores()) {
-      each.report(ps);
-      ps.println(StringUtil.separatorLine());
-    }
   }
 
   private List<MutationAnalysisUnit> buildMutationTests(

